orangecontrib/text/widgets/utils/widgets.py

```diff
@@ -351,11 +351,7 @@ def check_existence(self):
                 self.recent_files.remove(file)
 
     def open_file(self, path):
-        try:
-            self.on_open.emit(path if path != self.empty_file_label else '')
-        except (OSError, IOError):
-            self.loading_error_signal.emit('Could not open "{}".'
```

Comment:
Signal unused, doesn't seem to be defined anymore. Defer error handling to `on_open` handler.